issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -86,10 +84,6 @@ public String title() {
 
     @Override
     public void setTitle(String title) {
-        if (needSecurity) {
-            log.warning("Issue title does not support setting a security level - ignoring");
```

Comment:
Keep this, but change log message to "...setting a visibility role...".

Reply:
Ok, I removed this because I found it's never used. But yes, it's better to keep it